docgen/src/guide/Highlighting_results.md

```diff
@@ -73,13 +73,36 @@ Here is an example of a custom Highlight widget. It can be used the same
 way as the [widgets](guide/Highlighting_results.html#highlight-and-snippet-widgets).
 
 ```jsx
-const CustomHighlight = connectHighlight(({highlight, attributeName, hit}) => {
-  const parsedHit = highlight({attributeName, hit, highlightProperty: 'highlightProperty'});
-  return parsedHit.map(part => {
-    if(part.isHighlighted) return <mark>{part.value}</mark>;
-    return part.value;
-  });
-});
+import React from 'react';
+import { connectHighlight } from 'react-instantsearch/connectors';
+import { InstantSearch, Hits } from 'react-instantsearch/dom';
+
+const CustomHighlight = connectHighlight(
+  ({ highlight, attributeName, hit, highlightProperty }) => {
```

Comment:
What about putting here `highlightProperty = '_highlightResult’` Then it would be obvious to me.